### PR TITLE
LVM-408: Implement input masks for tutor form 

### PR DESCRIPTION
**Pull request by zack**
https://cs5500-jira.ccs.neu.edu/browse/LVM-408

* Also updates some tests and validations now that we have the
  support from the front-end.

### CODE REVIEW

File: spec/factories/tutor.rb [hunk 1]
```diff
@@ -20,6 +20,8 @@
     race { ApplicationHelper.race.sample[0] }
     smartt_id { Faker::Base.numerify('####-######') }
     state { ApplicationHelper.us_states.sample[0] }
+    training_date { Faker::Date.between(1.year.ago, 1.day.ago) }
```
Comment (xingyif) [inline spec/factories/tutor.rb]: 1 year ago? Are we sure no training can be older than 1 yr?

File: spec/factories/tutor.rb [hunk 2]
```diff
@@ -44,6 +46,8 @@
...
```
Comment (xingyif) [inline spec/factories/tutor.rb]: same qs here

File: spec/factories/tutor.rb (hunk 1 above)
Comment (zack) [reply spec/factories/tutor.rb]: It doesn't really matter. There's no validation around the date. But we can change it if you like.

Comment (xingyif) [reply spec/factories/tutor.rb]: i'll do `10.year.ago` to be safe

**Review by xingyif — APPROVED** [review]

**Pull request merged**